expo/scheduler/urls.py

```diff
@@ -66,4 +66,6 @@
         views.contact_info, name='contact_info'),
     url(r'^scheduler/contact_by_role/([-\w]+)/?$',
         views.contact_by_role, name='contact_by_role'),
+    url(r'^scheduler/export_calendar/?$',
+        views.export_calendar, name='export_calendar'),
```

Comment:
it looks like all the cool switches available in the view are not available in the urls.py.

I think it would be an easy add to put in the switches for conference, format, day, and event type here...  i see it as optional, though, since the ticket didn't require such flexibility.